contracts/extensions/OldRolesFactory.sol

```diff
@@ -0,0 +1,51 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity >=0.5.3;
+pragma experimental ABIEncoderV2;
+
+import "./../ColonyDataTypes.sol";
+import "./../IColony.sol";
+import "./../ColonyAuthority.sol";
+import "./ExtensionFactory.sol";
+import "./OldRoles.sol";
+
+
+contract OldRolesFactory is ExtensionFactory, ColonyDataTypes {
+  mapping (address => OldRoles) public deployedExtensions;	
+
+  function deployExtension(address _colony) external {
+    require(
+      ColonyAuthority(IColony(_colony).authority()).hasUserRole(msg.sender, 1, uint8(ColonyRole.Root)) == true, 
```

Comment:
Doesn't matter a ton (also this is merged already) but could also use the `hasUserRole` implemented in the Colony itself, which does basically the same thing. If we're going to have this require in every Factory it would simplify the code to:

```
require(
      IColony(_colony).hasUserRole(msg.sender, 1, ColonyRole.Root),
      "colony-extension-user-not-root"
)
```

Also you don't need the `== true` here, since the return value is already a boolean.